### PR TITLE
Add basic Elasticsearch Service template

### PR DESCRIPTION
**Pull request by farski**
Creates a simply Elasticsearch Service deployment template, with limited customization

### CODE REVIEW

**Comment by farski** [comment]
Closes #296

File: etc/elasticsearch.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS: "*"
+            Action: "es:*"
+            Resource: !Sub "arn:aws:es:${AWS::Region}:${AWS::AccountId}:domain/*"
+  # #     AdvancedOptions:
+  # #       String: String
+      EBSOptions:
+        EBSEnabled: !Ref EbsEnabled
```
Comment (cavis) [inline etc/elasticsearch.yml]: Is this really optional though?  In the web console, I can't even select "Instance Storage" - it's greyed out.

File: etc/elasticsearch.yml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    Type: String
+    AllowedValues:
+      - true
+      - false
+  EbsVolumeType:
+    Type: String
+    AllowedValues:
+      - standard
+      - gp2
+  EbsVolumeSize:
```
Comment (cavis) [inline etc/elasticsearch.yml]: Maybe worth noting that this isn't a straightforward number?  The web console has some notes that you should...

> multiply the EBS storage size per node by the number of data nodes in your cluster to calculate the total storage available to your cluster

File: etc/elasticsearch.yml (hunk 1 above)
Comment (farski) [reply etc/elasticsearch.yml]: The API docs make it seem very optional. For now I kept the parameter, but forced it to always be `true`

**Pull request merged**